docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+data:
+  image: busybox
+  volumes:
+     - /tmp/ivydata/ironmq:/ironmq/db
+     - /tmp/data/redis:/data/
+ironmq:
+  image: iron/mq
```

Comment:
Yup, reason why we are using them — see [docs](http://dev.iron.io/mq/3/on-premise/installation/)